app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    try:
+        for friend in friends:
+            try:
+                cafe.visit_cafe(friend)
+            except NotWearingMaskError:
+                masks_to_buy += 1
+        if masks_to_buy == 1:
+            return f"Friends should buy {masks_to_buy} mask"
+        elif masks_to_buy > 1:
+            return f"Friends should buy {masks_to_buy} masks"
+    except VaccineError:
+        return "All friends should be vaccinated"
+    else:
+        return f"Friends can go to {cafe.name}"
```

Comment:
```suggestion
        return "All friends should be vaccinated"
    return f"Friends can go to {cafe.name}"
```
`else` redundant in this case

Reply:
I agree thank you :)